src/v2/Components/v2/ArtworkFilter/__tests__/NewArtworkFilterMobileActionSheet.jest.tsx

```diff
@@ -0,0 +1,86 @@
+import { mount } from "enzyme"
+import React from "react"
+import { ArtworkFilterContextProvider } from "../ArtworkFilterContext"
+import { ArtworkFilterMobileActionSheet } from "../ArtworkFilterMobileActionSheet"
+import { ArtworkFilters } from "../ArtworkFilters"
+import { flushPromiseQueue } from "v2/DevTools"
+
+jest.mock("sharify", () => ({
+  data: {
+    ENABLE_NEW_ARTWORK_FILTERS: true,
+  },
+}))
```

Comment:
I created this new test file so that I could enable this feature flag without affecting existing tests. (Some tests make assertions about filters that are hidden when this feature flag is enabled, such as the `SizeFilter`)

Ideally I would be able to enable this feature flag in the `it` block that needs it, but I was only able to see this change take hold if I mocked Sharify outside of the top-level `describe` block.

Reply:
**non-blocking** — This seems fine, and I assume we'd merge these files once we retire the `ENABLE_NEW_ARTWORK_FILTERS` env var.

But fwiw I recall using this pattern to do a similar conditional based on env vars, but within the same spec file....

https://github.com/artsy/force/blob/283670bc5c885ea1b1c44e110a0ab89c3489550a/src/desktop/apps/fair/test/fairRedirection.jest.ts#L32-L37

Reply:
Thanks for finding this example. It looks like that snippet is mocking a config object that gets loaded into Sharify. Unfortunately, Sharify never loads that object (or any other data) during the test run. The only way that I have been able to add data to Sharify is by mocking it at the top of the test file.

I'll keep the two separate test files for now, and have written a post-launch clean up ticket [FX-2793](https://artsyproduct.atlassian.net/browse/FX-2793) to remember to merge them.